File: mobile/app/src/main/java/com/n7/quizbattle/home_rank/api/dtos/UserRankListItemDto.java
package com.n7.quizbattle.home_rank.api.dtos;

import com.google.gson.annotations.SerializedName;

public class UserRankListItemDto {
    @SerializedName("userId")
    private int userId;

    @SerializedName("avatarUrl")
    private String avatarUrl;

    @SerializedName("displayName")
    private String displayName;

    @SerializedName("rankScore")
    private int rankScore;

    @SerializedName("rank")
    private int rank;

    public int getUserId() {
        return userId;
    }

    public String getAvatarUrl() {
        return avatarUrl;
    }

    public String getDisplayName() {
        return displayName;
    }

    public int getRankScore() {
        return rankScore;
    }

    public int getRank() {
        return rank;
    }
}

File: server/Models/Overviews/DTOs/UserRankListItemDto.cs
namespace Models.Overviews.DTOs
{
    public class UserRankListItemDto
    {
        public int UserId { get; set; }
        public string AvatarUrl { get; set; }
        public string DisplayName { get; set; }
        public int RankScore { get; set; }
        public int Rank { get; set; }
        public int Level { get; set; }
    }
}
